core/src/main/java/com/netflix/conductor/core/reconciliation/WorkflowSweeper.java

```diff
@@ -137,18 +139,25 @@ void unack(WorkflowModel workflowModel) {
                                 (workflowModel.getWorkflowDefinition().getTimeoutSeconds() != 0)
                                         ? workflowModel.getWorkflowDefinition().getTimeoutSeconds()
                                                 + 1
-                                        : properties.getWorkflowOffsetTimeout().getSeconds();
+                                        : workflowOffsetTimeout;
                     }
                 } else {
                     postponeDurationSeconds =
                             (workflowModel.getWorkflowDefinition().getTimeoutSeconds() != 0)
                                     ? workflowModel.getWorkflowDefinition().getTimeoutSeconds() + 1
-                                    : properties.getWorkflowOffsetTimeout().getSeconds();
+                                    : workflowOffsetTimeout;
                 }
                 break;
             }
         }
         queueDAO.setUnackTimeout(
                 DECIDER_QUEUE, workflowModel.getWorkflowId(), postponeDurationSeconds * 1000);
     }
+
+    @VisibleForTesting
+    long workflowOffsetWithJitter(long workflowOffsetTimeout) {
```

Comment:
Can you add more details (comments) about how large the value of `jitter` can be?

Reply:
I've added few lines of comment in the code. Basically, the jitter will be +- 1/3 of workflowoffset value. For 2700 seconds offset, the range of the offset will be between 1800-3600

Reply:
M